examples/static/microphone-streaming-auto-stop.html

```diff
@@ -34,7 +34,6 @@
 
       var stream = WatsonSpeech.SpeechToText.recognizeMicrophone({
         token: token,
-//        continuous: false, // no longer supported
```

Comment:
@nfriedly sorry for the reply delay. I'll try to get an extra commit in this PR to address this issue today/tonight.